packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -204,7 +208,11 @@ export function drawLivenessOvalInCanvas({
     ctx.clearRect(0, 0, canvasWidth, canvasHeight);
 
     // fill the canvas with a transparent rectangle
-    ctx.fillStyle = 'rgba(255, 255, 255, 1.0)';
+    ctx.fillStyle = getComputedStyle(canvas).getPropertyValue(
+      isStartScreen
+        ? '--amplify-colors-background-primary'
+        : '--amplify-colors-white'
+    );
```

Comment:
This is for a canvas so we set the style here, we also don't have a classname set for this to my knowledge. This is just so that we can resolve the proper color from the theme